apps/web/src/services/administration/iam/user/type.ts

```diff
@@ -112,3 +132,9 @@ export interface ChannelItem {
 	notification_level?: string;
 	protocol_name: string;
 }
+
+export const PasswordType = Object.freeze({
+    KEEP: 'KEEP',
+    RESET: 'RESET',
+    MANUALLY: 'MANUALLY',
+}as const);
```

Comment:
nitpick: We decided to use only `as const` [before](https://www.notion.so/spaceone-io/constant-22cee41a115e47a1a277f3a936f275b9?pvs=4). (sorry I didn't tell you in advance😅)

```suggestion
export const PasswordType = {
    KEEP: 'KEEP',
    RESET: 'RESET',
    MANUALLY: 'MANUALLY',
} as const;
```